toolchains/docker/pull.bzl.tpl

```diff
@@ -0,0 +1,12 @@
+load("@io_bazel_rules_docker//container:pull.bzl",
+    _container_pull="container_pull")
+
+# Simply call the regular container_pull with the docker client config directory
```

Comment:
S/ Simply call the regular container_pull with ...
/ Call container_pull with ...

Reply:
Done